src/sentry/models/releasefile.py

```diff
@@ -99,14 +108,22 @@ def cache_path(self):
         return options.get("releasefile.cache-path")
 
     def getfile(self, releasefile):
+        return self.get_regular_file(releasefile.file, releasefile.organization_id)
+
+    def get_regular_file(self, file_: File, organization_id: int) -> FileObj:
```

Comment:
Let's make the archive a regular release file, too. This avoids a special case for caching, and only adds +1 release file, which is acceptable. Then, you can back this out.